stable/redis/templates/svc-external-access.yaml

```diff
@@ -0,0 +1,46 @@
+{{- if .Values.cluster.externalAccess.enabled }}
+{{- $fullName := include "redis.fullname" . }}
+{{- $nodesCount := .Values.cluster.nodes | int }}
+{{- $root := . }}
+
+{{- range $i, $e := until $nodesCount }}
+{{- $targetPod := printf "%s-%d" (printf "%s" $fullName) $i }}
+
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: {{ template "redis.fullname" $ }}-{{ $i }}-svc
+  labels: {{- include "redis.labels" $ | nindent 4 }}
+    pod: {{ $targetPod }}
+  annotations: {{ include "redis.tplValue" ( dict "value" $root.Values.cluster.externalAccess.service.annotations "context" $) | nindent 4 }}
+spec:
+  type: {{ $root.Values.cluster.externalAccess.service.type }}
+  {{- if eq $root.Values.cluster.externalAccess.service.type "LoadBalancer" }}
+  {{- if $root.Values.cluster.externalAccess.service.loadBalancerIP }}
+  loadBalancerIP: {{ index $root.Values.cluster.externalAccess.service.loadBalancerIP $i }}
+  {{- end }}
+  {{- end }}
+  ports:
+    - name: redis
+      {{- if eq $root.Values.cluster.externalAccess.service.type "LoadBalancer" }}
+      port: {{ $root.Values.cluster.externalAccess.service.port }}
+      {{- else }}
+      port: {{ index $root.Values.cluster.externalAccess.service.nodePort $i }}
+      {{- end }}
+      {{- if $root.Values.cluster.externalAccess.service.nodePort }}
+      nodePort: {{ index $root.Values.cluster.externalAccess.service.nodePort $i }}
+      {{- end }}
+      {{- if eq $root.Values.cluster.externalAccess.service.type "LoadBalancer" }}
+      targetPort: redis
+      {{- else }}
+      targetPort: {{ index $root.Values.cluster.externalAccess.service.nodePort $i }}
+      {{- end }}
+    - name: redis-cluster
```

Comment:
Change it to tcp-redis-cluster or udp-redis-cluster (not sure which of the two apply) or check if the protocol in the bus is the same as the standard redis port.

Reply:
I guess it is TCP:
> To perform their tasks all the cluster nodes are connected using a TCP bus and a binary protocol, called the Redis Cluster Bus